core/arch/arm/plat-imx/platform_config.h

```diff
@@ -61,22 +61,22 @@
 #define CFG_DDR_START			DDR_PHYS_START
 #define CFG_DDR_SIZE			DDR_SIZE
 
-#ifndef CFG_DDR_TEETZ_RESERVED_START
-#define CFG_DDR_TEETZ_RESERVED_START	0x9E000000
+#ifndef CFG_TZDRAM_BASE
+#define CFG_TZDRAM_BASE			0x9E000000
 #endif
 
-#define CFG_DDR_TEETZ_RESERVED_SIZE	0x02000000
-
-#define CFG_PUB_RAM_SIZE	(2 * 1024 * 1024)
+#ifndef CFG_TZDRAM_SIZE
+#define CFG_TZDRAM_SIZE			(30 * 1024 * 1024)
+#endif
```

Comment:
```
#if !defined(CFG_TZDRAM_SIZE) && defined(CFG_DDR_TEETZ_RESERVED_SIZE)
#define CFG_TZDRAM_SIZE			CFG_DDR_TEETZ_RESERVED_SIZE
#endif
#ifndef CFG_TZDRAM_SIZE
#define CFG_TZDRAM_SIZE			0x02000000  // or 32 * 1024 * 1024  There are some SZ_*MB macro somewhere.
#endif
```

Reply:
`CFG_DDR_TEETZ_RESERVED_SIZE` was not previously configurable, so in the interest of standardizing on `CFG_TZDRAM_SIZE`, I have not restored `CFG_DDR_TEETZ_RESERVED_SIZE`.